crates/polars-arrow/src/bitmap/immutable.rs

```diff
@@ -63,6 +64,10 @@ pub struct Bitmap {
     unset_bit_count_cache: AtomicU64,
 }
 
+fn has_cached_unset_bit_count(ubcc: u64) -> bool {
```

Comment:
Can we tag this one with `inline`?

Reply:
Functions in the same crate will get automatically inlined. But I have added `#[inline(always)]` to force LLVM to always do that.